truenas/webui
# NAS-114558 / None / Replaced IxCombobox with IxCombobox2

Test
1. Check LLDP service config form
2. Check Localization form
3. Check UPS service form

Check `AsyncSimpleComboboxProvider`
